Profile Folder/chrome/locale/uk/properties/gsettings.properties

```diff
@@ -1,15 +1,15 @@
-geckiumSettings=[!!! Gèçƙïú₥ §èƭƭïñϱƨ !!!]
+geckiumSettings=Налаштування Geckium
 
 addAnApp=Додати дотаток
 
 themeHasNoDescription=Ця тема не має опису.
-builtInTheme=[!!! [ßÛÌ£T-ÌN] {{themeName}} !!!]
-showInFolder=[!!! §λôω ïñ ƒôℓδèř !!!]
-apply=[!!! Âƥƥℓ¥ !!!]
-yes=[!!! Ýèƨ !!!]
+builtInTheme=Вбудований {{themeName}}
```

Comment:
`builtInTheme` seems to be `Built-in {{theme}}`, I think it should probably just be `[BUILT-IN] {{themeName}}`.
Here's another example of how it looks like in english:
![image](https://github.com/user-attachments/assets/7394a0ab-f886-46b8-b516-88410df1fa2a)
What do you think?